### PR TITLE
[GEOT-5635] ImageMosaic loading create one object blocking the finalization thread for a while

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

File: modules/plugin/imagemosaic/pom.xml [hunk 1]
```diff
@@ -207,20 +207,4 @@
   </dependencies>
 
 
-  <build>
```
Comment (aaime) [inline modules/plugin/imagemosaic/pom.xml]: I was not sure about this one, just noticed it makes the build slower and harder to debug

Comment (aaime) [inline modules/plugin/imagemosaic/pom.xml]: Maybe someone on Windows can try the build and see if removing this causes issues? I've checked the history, but it has been there longer than what we have in git

**Comment by dromagnoli** [comment]
Looks good to me.

File: modules/plugin/imagemosaic/pom.xml (hunk 1 above)
Comment (dromagnoli) [inline modules/plugin/imagemosaic/pom.xml]: Not sure about this, neither.

Comment (dromagnoli) [inline modules/plugin/imagemosaic/pom.xml]: I have built the imageMosaic module on Windows (on top of your aaime:replaceSerializableRenderedImage) and I had no issues.

**Pull request merged**